### PR TITLE
Sunderland: Update theme and prepare for Dotorg submission

### PR DESCRIPTION
**Pull request by beafialho**
**Changes from WordPress Playground**

I made some improvements to the theme before submitting it to Dotorg: 

- Added image from theme screenshot license
- Rearranged the templates (spacings mostly)
- Did some fine-tuning in Global Styles, font size presets

### CODE REVIEW

File: sunderland/patterns/home.php [hunk 1]
```diff
@@ -6,8 +6,8 @@
...
-<div class="wp-block-cover has-contrast-color has-text-color" style="padding-right:5vw;padding-left:5vw;min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-base-background-color has-background-dim-10 has-background-dim"></span><img class="wp-block-cover__image-background" alt="" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/summertime_image.webp" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:group {"align":"full","style":{"dimensions":{"minHeight":"100vh"},"spacing":{"padding":{"right":"0vw","left":"0vw"}}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","verticalAlignment":"space-between"}} -->
+<!-- wp:cover {"url":"https://playground.wordpress.net/scope:0.1783832307829636/wp-content/themes/sunderland/assets/images/summertime_image.webp","dimRatio":10,"overlayColor":"base","isUserOverlayColor":true,"minHeight":100,"minHeightUnit":"vh","lock":{"move":false,"remove":true},"style":{"spacing":{"padding":{"right":"5vw","left":"5vw"}}},"textColor":"contrast","layout":{"type":"default"}} -->
```
Comment (beafialho) [inline sunderland/patterns/home.php]: This template shouldn't be updated. When I was editing the theme, the background photo disappeared, likely due to the localisation of the photo but the previous code was working fine.

File: sunderland/theme.json [hunk 2]
```diff
@@ -1251,17 +952,17 @@
...
 			"padding": {
 				"bottom": "0rem",
-				"left": "0rem",
-				"right": "0rem",
+				"left": "5vw",
+				"right": "5vw",
 				"top": "0rem"
 			}
 		},
 		"typography": {
```
Comment (beafialho) [inline sunderland/theme.json:597]: I could not change this myself in theme.json but we should neither use or bundle the font Helvetica with the theme, we should just specify Helvetica as a system font, just like the theme "Rainfall" does this.

File: sunderland/patterns/home.php (hunk 1 above)
Comment (mikachan) [reply sunderland/patterns/home.php]: I've updated the image path so it's relative to the theme again, and I can see the cover image locally.

File: sunderland/theme.json (hunk 2 above)
Comment (mikachan) [reply sunderland/theme.json:597]: I've removed Helvetica as a bundled font here: https://github.com/Automattic/themes/pull/7812/commits/4535d77b5955797767c93382bf4565515ca18167

**Review by mikachan — APPROVED** [review]
Thanks @beafialho! I've addressed your comments and also updated the footer pattern to match the new block markup from the pattern in the editor. This is looking good to bring in.

**Pull request merged**